src/frontend/apps/standalone_site/src/components/Layout/MainLayout.tsx

```diff
@@ -1,5 +1,5 @@
-import { Nullable, colorsTokens } from 'lib-common';
-import { Box, BoxProps } from 'lib-components';
+import { Breakpoints, Nullable, colorsTokens } from 'lib-common';
```

Comment:
In the commit title, replace `devise` by `device`

Reply:
https://github.com/openfun/marsha/pull/2499/commits/2a397954fc4a333036ca20390d2f49f34ecb265d